examples/edge30-full-example/custom/files/basic-setup.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Pre-requisites. Cluster already running
+export KUBECTL="/var/lib/rancher/rke2/bin/kubectl"
+export KUBECONFIG="/etc/rancher/rke2/rke2.yaml"
+
+##################
+# METAL3 DETAILS #
+##################
+export METAL3_CHART_TARGETNAMESPACE="metal3-system"
+export METAL3_CLUSTERCTLVERSION="1.6.2"
+export METAL3_CAPICOREVERSION="1.6.0"
```

Comment:
@e-minguez should be 1.6.2 ??? -> "cluster-api:v1.6.2" --infrastructure "metal3:v1.6.0"

Reply:
Done, thanks.